src/loadData.ts

```diff
@@ -0,0 +1,13 @@
+import { Todo, User } from './types';
+
+
+const todosUrl = 'https://jsonplaceholder.typicode.com/todos';
+const usersUrl = 'https://jsonplaceholder.typicode.com/users';
```

Comment:
```suggestion
const API_URL = 'https://jsonplaceholder.typicode.com/';
```

где нужно дописывай `todos` / `users`